### ISSUE TITLE
Deprecated symbol names?

### ISSUE THREAD

**Issue by LilaQ** [SFSafeSymbols/SFSafeSymbols#97]
Hi, I'm using SFSafeSymbols in my new project, but I get +100 Warnings in my log that several names were deprecated / changed in iOS 14. 

Is there a way to get a release that's for iOS 14+ ? 
I installed through CocoaPods btw.

Thanks in advance

**Comment by knothed** [comment]
Hi @LilaQ, thanks for raising this issue!

Just to clarify: Are you seeing compile-time warnings or runtime warnings (i. e. in the log)? Could you maybe provide a screenshot that shows the warnings that you are seeing?

**Comment by abdelmagied94** [comment]
Hi @knothed,

I'm also facing this issue. I'm using the package in a fresh project with a deployment target `iOS 14.0`.

<img width="1181" alt="Screen Shot 2022-04-06 at 11 58 25 PM" src="https://user-images.githubusercontent.com/48062713/162078948-4b03ba20-37e2-44a2-a4f2-a328fef7d5ab.png">

**Comment by fredpi** [comment]
This problem is probably already fixed in the current `stable` and a release will be out soon.

Maybe you can give the current `stable` a try to test whether it really works, @abdelmagied94?

**Comment by abdelmagied94** [comment]
@fredpi Thank you for your response 🙏  I found the issue that I delete the `IPHONEOS_DEPLOYMENT_TARGET` from pods projects as follows:
```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings.delete 'IPHONEOS_DEPLOYMENT_TARGET'
    end
  end
end
```

**Comment by StevenSorial** [comment]
@abdelmagied94 if your deployment target is 14.0, then you can get rid of the warnings by applying the fix-it and upgrading to the new names. If I understood correctly this behavior is intended ([read](https://github.com/SFSafeSymbols/SFSafeSymbols/pull/80#issuecomment-974787421)) because its showing upgrades that are now possible because you have 14.0 as the minimum requirement.

@fredpi please correct me if I got it wrong

**Comment by fredpi** [comment]
@StevenMagdy I also thought that at first, but in fact these are warnings from the library itself – to be precise: from the `allCases` array. In previous versions, we would include deprecated symbols in that array and that is now fixed.

**Comment by StevenSorial** [comment]
I should've looked at the filename before answering 😄 

@abdelmagied94 yeah that is fixed on the `stable` branch, you could direct CocoaPods to it. You can also do like me and silence all warnings from all cocoapods dependencies using `inhibit_all_warnings` 😉

**Comment by fredpi** [comment]
This should now be fixed with the [3.3 release](https://github.com/SFSafeSymbols/SFSafeSymbols/releases/tag/3.3). Feel free to reopen, if you are still getting these warnings.
